packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1389,6 +1397,52 @@ export class Profiles extends ProfilesCache {
         return;
     }
 
+    private async checkExistingConfig(filePath: string) {
+        let found = false;
+        let location: string;
+        const existingLayers = await this.getConfigLayers();
+        for (const file of existingLayers) {
+            if (file.path.includes(filePath)) {
+                found = true;
+                const createButton = localize("checkExistingConfig.createNew.button", "Create New");
+                const message = localize(
+                    "checkExistingConfig.createNew.message",
+                    `A Team Configuration File already exists in this location\n{0}\nWould you like to continue with creating a new file?`,
```

Comment:
Should we mention something about the behavior of `zowe config init` ?
Right now it feels like we are about to replace the entire file. 😋